scss/_root.scss

```diff
@@ -21,6 +21,7 @@
     --#{$prefix}#{$color}-rgb: #{$value};
   }
 
+  --#{$prefix}accessible-orange: #{$accessible-orange}; // Boosted mod
```

Comment:
In Bootstrap dark mode PR, `--bs-{themeColor}-text` is introduced:

```css
--bs-primary-text: #0a58ca;
--bs-secondary-text: #6c757d;
--bs-success-text: #146c43;
--bs-info-text: #087990;
--bs-warning-text: #997404;
--bs-danger-text: #b02a37;
--bs-light-text: #6c757d;
--bs-dark-text: #495057;
```

Not 100% sure yet that it's going to be merged like this but it seems likely. I have the feeling that having `--bs-primary-text` would be more appropriate here.

In light mode `--bs-primary-text` would be `$accessible-orange` and in dark mode (dark variant for Boosted), it would be `$brand-orange`. And so, everywhere in the code we use something without the word "accessible" and not linked to a specific color; we can't do that `--#{$prefix}accessible-orange: #{$brand-orange};` IMHO. It would be like saying `--bs-danger`: #($warning)`.

What do you think?

Reply:
Agree on this, changed on last commit ! Forgot about checking the Bootstrap one 🤦